src/activities/ballcatch/Ballcatch.qml

```diff
@@ -198,9 +200,11 @@ ActivityBase {
 
         Image {
             id: leftShift
-            x: background.width / 4 - width 
-            y: rightHand.y - height / 2
+            x: background.width / 4 - width
+            y: ((background.vert) ? rightHand.y - height : rightHand.y - height / 2)
             source: "qrc:/gcompris/src/activities/ballcatch/resource/arrow_key.svg"
+            scale: ((background.vert) ? 0.75 : 1.0)
```

Comment:
no need to have parenthesis when the expression is simple

Reply:
i will remember this for future changes. thanks.